themes/default/content/what-is/resolve-list-buckets-invalid-access-key-id.md

```diff
@@ -0,0 +1,110 @@
+---
+title: An error occurred (InvalidAccessKeyId) when calling the ListBuckets operation
+allow_long_title: true
+meta_desc: |
+     Use Pulumi ESC and dynamic credentials to run commands like aws ListBuckets in a more secure and seamless way.
+type: what-is
+page_title: An error occurred (InvalidAccessKeyId) when calling the ListBuckets operation
+---
+
+The error message "An error occurred (InvalidAccessKeyId) when calling the ListBuckets operation" in AWS (Amazon Web Services) indicates that the provided AWS Access Key Id does not exist in their records. AWS Access Key Ids are used by AWS to authenticate and authorize access to various AWS services and resources. When you attempt an AWS CLI operation with an invalid or nonexistent Access Key Id, this error arises, as AWS cannot validate the provided identifier.
+
+[Pulumi ESC (Environments, Secrets, and Configurations)](/docs/pulumi-cloud/esc/) offers a solution to the challenges of managing credentials and can make errors like this a thing of the past. By enabling the [management of dynamic credentials from AWS using OIDC](/blog/esc-env-run-aws/), Pulumi ESC simplifies and secures your AWS CLI operations. This approach eliminates the need for manually providing credentials and is a more secure solution than the use of long-term credentials, which can often present a security risk. Pulumi ESC also enables you to focus on your tasks without the interruption of credential-related errors, providing a more efficient flow for your AWS operations and tasks.
```

Comment:
should this one be adjusted so there is additional context connected to the ErrorCode? Like we can say instead of using Access Keys, you can use dyanmically generated ones that are more secure, etc, etc